### PR TITLE
Update operatorhub PR template before submitting update for 1.15

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@jpkrohling One of the additional questions is whether the PR is signed - not sure how we can do this via automation?

File: .ci/operatorhub.sh [hunk 1]
```diff
@@ -14,6 +14,7 @@ OLD_PWD=$(pwd)
 VERSION=$(grep operator= versions.txt | awk -F= '{print $2}')
 PKG_FILE=deploy/olm-catalog/jaeger-operator/jaeger-operator.package.yaml
 CSV_FILE=deploy/olm-catalog/jaeger-operator/${VERSION}/jaeger-operator.v${VERSION}.clusterserviceversion.yaml
+CRD_FILE=deploy/crds/jaegertracing.io_jaegers_crd.yaml
```
Comment (objectiser) [inline .ci/operatorhub.sh]: Currently there are two `_crd.yaml` files in that folder, one with the `v1` version in the name. Which one should be copied? Maybe we need to standardise on a naming convention and use it in the manual deploy instructions aswell - i.e. we might have multiple versions at some point - are they all going to be included in the same (non versioned) crd file, or in separate files?

**Comment by objectiser** [comment]
Tried testing the operator (olm-catalog/jaeger-operator) locally, using instructions from https://github.com/operator-framework/community-operators/blob/master/docs/using-scripts.md#deploying-and-testing-your-operator, but it complained about metadata annotations size - which I eventually got rid of by removing the icon.

It might be good if we could include the
```
make operator.install OP_PATH=upstream-community-operators/jaeger
```
as part of the operatorhub release process, to ensure it is valid before submitting the PR.

File: .ci/operatorhub.sh (hunk 1 above)
Comment (jpkrohling) [reply .ci/operatorhub.sh]: I just tried creating a new operator, and newer versions of the operator-sdk generate a file following the pattern without the version, with a pluralized kind name (`jaegertracing.io_jaegers_crd.yaml`). We should be able to remove `jaegertracing_v1_jaeger_crd.yaml` then.

File: .ci/operatorhub.sh [hunk 2]
```diff
@@ -34,7 +35,9 @@ git rebase -q upstream/master
 
 for dest in upstream-community-operators community-operators; do
     cp "${OLD_PWD}/${PKG_FILE}" "${COMMUNITY_OPERATORS_REPOSITORY}/${dest}/jaeger/${DEST_PKG_FILE}"
-    cp "${OLD_PWD}/${CSV_FILE}" "${COMMUNITY_OPERATORS_REPOSITORY}/${dest}/jaeger/${DEST_CSV_FILE}"
+    mkdir "${COMMUNITY_OPERATORS_REPOSITORY}/${dest}/jaeger/${VERSION}"
```
Comment (jpkrohling) [inline .ci/operatorhub.sh]: Could you add a `-p` to the command? I think it might fail if the directory exists already, and won't fail if `-p` is passed.

**Comment by jpkrohling** [comment]
> One of the additional questions is whether the PR is signed - not sure how we can do this via automation?

Signed, or signed off? It should be possible to do it either way, just need to expose the right secrets to the job, in order to setup the key used for the signing.

In any case, I'm not sure if that level of automation would be beneficial to us, as I have a feeling that we might need to update the PR template quite often. How about we change the release instructions to mention that this command should be executed manually, and only after checking whether the template is up to date (this last step should be possible to do in the script itself...)

**Comment by jpkrohling** [comment]
This PR is ready to be merged, but it does *not* include the testing command, as I couldn't get it to work for me:

```
$ make operator.install OP_PATH=upstream-community-operators/jaeger
```

I tried it with both Docker and Podman, without luck.

**Comment by jpkrohling** [comment]
@objectiser could you please have another look? There are a couple of things I adjusted:

1. commit sign-off 
1. the image version in the CSV was wrong (1.14.0 vs. 1.15.0, need to check why the release script didn't bump it)
1. duplicate entry in the "crds" section -- didn't cause any harm, and I think it's caused by the fact that there were two similar files there (jaegertracing.io_jaegers_crd.yaml and jaegertracing_v1_jaeger_crd.yaml)

The following PRs are the result of this script:

https://github.com/operator-framework/community-operators/pull/891
https://github.com/operator-framework/community-operators/pull/892

I ran the tests manually, following this doc: 
https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#testing-operator-deployment-on-kubernetes

And here are the logs for the operator that got installed:

```
$ kubectl logs jaeger-operator-69fd677b5-hzf2l
time="2019-11-14T15:05:04Z" level=info msg=Versions arch=amd64 identity=marketplace.jaeger-operator jaeger=1.15.0 jaeger-operator=v1.15.0 operator-sdk=v0.11.0 os=linux version=go1.12.10
time="2019-11-14T15:05:05Z" level=info msg="Auto-detected the platform" platform=kubernetes
time="2019-11-14T15:05:05Z" level=info msg="Automatically adjusted the 'es-provision' flag" es-provision=no
time="2019-11-14T15:05:05Z" level=info msg="Automatically adjusted the 'kafka-provision' flag" kafka-provision=no
time="2019-11-14T15:05:05Z" level=info msg="The service account running this operator does not have the role 'system:auth-delegator', consider granting it for additional capabilities"
time="2019-11-14T15:05:06Z" level=warning msg="could not generate and serve custom resource metrics" error="discovering resource information failed for Elasticsearch in logging.openshift.io/v1: apiVersion logging.openshift.io/v1 and kind Elasticsearch not found available in Kubernetes cluster"
time="2019-11-14T15:05:07Z" level=warning msg="Could not create ServiceMonitor object" error="no ServiceMonitor registered with the API"
time="2019-11-14T15:05:07Z" level=warning msg="Install prometheus-operator in your cluster to create ServiceMonitor objects" error="no ServiceMonitor registered with the API"
```

**Pull request merged**